### DISCUSSION TITLE
How to learn php?

### DISCUSSION THREAD

**Discussion by LearnerDavid** [community/community#19689] [Other Feature Feedback, Questions, & Ideas]
How to learn php?
What are the recommended websites to learn php?

**Comment by Elton-Ribeiro** [comment]
Hi @LearnerDavid!

If you've never studied PHP before, maybe you could try a udemy class as a first step in your learning.

**Comment by melvincwng** [comment]
Hi @LearnerDavid, YouTube, Codecademy, W3School are all excellent resources to learn php, or any programming language (in general).

Can consider Udemy courses as well, as already suggested by @Elton-Ribeiro.

All the best in your learning journey 😊

**Reply by melvincwng** [reply]
@LearnerDavid and @Elton-Ribeiro , if you guys are interested, let's connect & follow on GitHub :)

Trying to expand my network of developers on GitHub as well as learn new skills too together ^^

**Comment by cupid20103** [comment]
Hi, @LearnerDavid.
If you want to learn PHP, please check this link.
https://www.w3schools.com/php/default.asp
This is PHP course of w3school.

**Comment by mrcsxsiq** [comment]
Hi @LearnerDavid,

You can try to learning by youtube channels or udemy but maybe [this repository](https://github.com/ziadoz/awesome-php) could be useful for you too

**Comment by Zoltus** [comment]
@LearnerDavid I recommend using sololearn! Its free and u can have it on mobile and pc.
It has courses and tests and has some fun reward system.
[https://www.sololearn.com/home](url)

**Comment by MThos** [comment]
W3Schools to get a basic understanding. Udemy course when on sale for <$20 can be an excellent value if you want a more detailed full length tutorial, but I would check W3 first to see if PHP is something you actually are interested in learning before spending money.
